keps/sig-testing/2291-presubmit-config-inside-the-tested-repo /README.md

```diff
@@ -1,22 +1,3 @@
----
```

Comment:
same nit as https://github.com/kubernetes/enhancements/pull/2292/files#r560372201

Reply:
Thanks for the review. I think there is no space in folder name here.

Reply:
Ah! I think I meant to comment on the rename for the above one only. Thank you for fixing that! :)